### DISCUSSION TITLE
heading icons

### DISCUSSION THREAD

**Discussion by tjodalv** [tailwindlabs/heroicons#1236] [Ideas]
Hi there,

First, I want to express my gratitude for the beautifully designed icons in Heroicons. I have an idea/request to share: while there are heading icons for H1, H2, and H3, I noticed that there aren’t options for H4, H5, H6, or a generic "H" icon. Could these be considered for a future release?

Thank you again for your amazing work!

**Comment by ndri** [comment]
I've added `h4`, `h5`, `h6` and `heading` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here:
- https://sidekickicons.com/?icon=h4
- https://sidekickicons.com/?icon=h5
- https://sidekickicons.com/?icon=h6
- https://sidekickicons.com/?icon=heading
